packages/autocomplete-plugin-algolia-insights/src/createAlgoliaInsightsPlugin.ts

```diff
@@ -190,3 +224,23 @@ function getOptions(options: CreateAlgoliaInsightsPluginParams) {
     ...options,
   };
 }
+
+function loadInsights(environment: typeof window) {
+  const errorMessage = `[Autocomplete]: Could not load search-insights.js. Please load it manually following https://alg.li/insights-autocomplete`;
+
+  try {
+    const script = environment.document.createElement('script');
+    script.async = true;
+    script.src = ALGOLIA_INSIGHTS_SRC;
+
+    script.onerror = () => {
+      // eslint-disable-next-line no-console
+      console.error(errorMessage);
+    };
+
+    document.body.appendChild(script);
+  } catch (cause) {
+    // eslint-disable-next-line no-console
+    console.error(errorMessage);
+  }
```

Comment:
Not a huge fan of using `console.error`s but since there's no global error handler in Autocomplete, throwing would be hardly catchable for users.

I'd rather use those now (Sentry lets you catch calls to the Console API) and work on an `onError` handler in a separate PR to centralize error handling.